examples/lit-ts/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "example-lit-ts",
+  "private": true,
+  "version": "0.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "lit": "^2.2.2",
+    "rollup-plugin-postcss-lit": "^2.0.0"
+  },
+  "scripts": {
+    "storybook": "start-storybook -p 6005",
+    "build-storybook": "build-storybook",
+    "preview-storybook": "http-server storybook-static --port 6005 --silent",
+    "test": "wait-on tcp:6005 && test-storybook --url 'http://localhost:6005'",
+    "test-ci": "run-p --race test preview-storybook"
+  },
+  "author": "",
+  "license": "MIT",
+  "devDependencies": {
+    "@babel/core": "^7.17.9",
+    "@mdx-js/react": "^1.6.22",
+    "@storybook/addon-actions": "^6.4.22",
```

Comment:
Can you please update these to the same versions used in the other examples?  If they get out-of-sync, multiple copies of various storybook modules get installed in the monorepo, and it causes a mess.